### PR TITLE
Add scenrios for DTD-496 to make interest rate optional for non inter…

### PR DESCRIPTION
**Pull request by rogerball**
…est bearing debts

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ifs/InterestForecastingSteps.scala [hunk 1]
```diff
@@ -362,4 +362,13 @@ class InterestForecastingSteps extends ScalaDsl with EN with Eventually with Mat
   Given("debt payment plan details") { (dataTable: DataTable) =>
     createPaymentPlanRequestBody(dataTable)
   }
+
+  Then("the ([0-9])(?:st|nd|rd|th) debt summary will not have any calculation windows") { (summaryIndex: Int) =>
```
Comment (vivekhmrc) [inline src/test/scala/uk/gov/hmrc/test/api/cucumber/stepdefs/ifs/InterestForecastingSteps.scala:366]: We are still discussing this if this is the right way to go and need to discuss with Helen and Chris about this specific scenario.

File: src/test/resources/features/ifs/MultipeDebtItems.feature [hunk 2]
```diff
@@ -29,10 +29,8 @@ Feature: Multiple Debt Items
     And the 1st debt summary will contain
       | interestBearing | numberChargeableDays | interestDueDailyAccrual | interestDueDutyTotal | unpaidAmountDuty | totalAmountIntDuty | amountOnIntDueDuty |
       | false           | 0                    | 0                       | 0                    | 400000           | 400000             | 400000             |
-    And the 1st debt summary will have calculation windows
-      | periodFrom | periodTo   | numberOfDays | interestRate | interestDueDailyAccrual | interestDueWindow | amountOnIntDueWindow | unpaidAmountWindow |
-      | 2018-12-16 | 2019-02-03 | 0            | 0.0          | 0                       | 0                 | 100000               | 100000             |
-      | 2018-12-16 | 2019-04-14 | 0            | 0.0          | 0                       | 0                 | 400000               | 400000             |
+    And the 1st debt summary will not have any calculation windows
```
Comment (vivekhmrc) [inline src/test/resources/features/ifs/MultipeDebtItems.feature:32]: I know that this is the proposed solution but we need to agree if this is the right solution with Chris and Helen.

**Review by vivekhmrc — APPROVED** [review]
The Tests are good to merge after we have agreed to the calculation windows logic. Thanks

**Comment by vivekhmrc** [comment]
LGTM 👍

**Review by thadeusssewanyana — APPROVED** [review]

**Pull request merged**